packages/rke2-ingress-nginx/package.yaml

```diff
@@ -0,0 +1,2 @@
+url: https://kubernetes-charts.storage.googleapis.com/nginx-ingress-1.36.3.tgz
```

Comment:
If you rename this, you might have to make sure the chart.yaml's name is also renamed to rke2-ingress-nginx